### PR TITLE
docs(EmptyQwenImageLayeredLatentImage): explain the layers + 1 slot

### PR DESCRIPTION
**Pull request by christian-byrne**
## Problem

`EmptyQwenImageLayeredLatentImage` allocates `torch.zeros([batch_size, 16, layers + 1, height // 8, width // 8])` (`comfy_extras/nodes_qwen.py:109-130`), and the docs never say what the `+ 1` is. The `layers` description currently reads:

> The number of additional layers to add to the latent structure. This defines the depth of the latent representation.

That is not just incomplete, it points the reader the wrong way: it suggests the extra slot is structural padding, when it is an actual image the model produces. A core maintainer could not work these semantics out from the code, and nothing in core, on docs.comfy.org, or on the model card states them.

## What is actually true

comfyanonymous, on the semantics of this model:

> it's a model specific thing, it's because this model is an image generation model that regenerates the full image and the layers
>
> so if you set layers to zero you get only the original image
>
> but if you set layers to 2 you get the original image + 2 layers
>
> the first image is exactly the same as the original and should be discarded
>
> and all the layers when composited together should recreate the original image

So:

1. **Slot 0 is the full image, not a layer**, and should be discarded when only the layers are wanted.
2. **Compositing all the layers recreates the full image.** That is the reconstruction guarantee, and it is a usable correctness check.
3. **Layer order is positional and nothing carries it.** The layers ride the temporal axis of the 5D latent, the same axis video models use for frames. `LatentCutToBatch` with `dim="t"` (`comfy_extras/nodes_latent.py:217-256`) moves that axis into the batch dimension, after which a layer's z-order is just its index in the batch. There is no z-index field and no per-layer metadata anywhere in the pipeline, so reordering or filtering the batch reorders the layers.
4. `layers` has `min=0`, and `layers=0` still yields one image.
5. The Qwen-Image-Layered VAE decodes to 4 channels: its `encoder.conv1.weight` is `[96, 4, 3, 3, 3]`, so `comfy/sd.py:791` sets `output_channels = 4`. Decoded layers carry alpha.

## Change

English source only, in `comfyui_embedded_docs/docs/EmptyQwenImageLayeredLatentImage/en.md`:

- Rewrote the `layers` row so it states the `layers + 1` output count and that `0` is valid.
- Added a **Why there is one more slot than layers** section covering the five points above.
- Left the `Outputs` table, the AI-generated footer and the source fingerprint untouched.

## Other locales

Not touched, deliberately. `.cursorrules` in this repo says:

> Do not automatically start translation work for other language files unless actively requested by the user

The other 11 locales (`ar`, `es`, `fa`, `fr`, `ja`, `ko`, `pt-BR`, `ru`, `tr`, `zh`, `zh-TW`) all carry the same wrong `layers` description and need the same fix. Rather than machine-translate them badly, I have filed a tracking issue: https://github.com/Comfy-Org/embedded-docs/issues/114

## Related, not fixed here

comfyanonymous also said *"our qwen layered template is slightly wrong"*. `templates/image_qwen_image_layered.json` in `Comfy-Org/workflow_templates` runs `EmptyQwenImageLayeredLatentImage` with `layers=2` straight into `LatentCutToBatch(dim="t")` -> `VAEDecode` -> `SaveImage`, so it saves 3 images and the first is a duplicate of the input. That is a separate repo and a separate fix.

### CODE REVIEW

File: comfyui_embedded_docs/docs/EmptyQwenImageLayeredLatentImage/en.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Empty Qwen Image Layered Latent
 
-The Empty Qwen Image Layered Latent node creates a blank, multi-layered latent representation for use with Qwen image models. It generates a tensor filled with zeros, structured with a specified number of layers, batch size, and spatial dimensions. This empty latent serves as a starting point for subsequent image generation or manipulation workflows.
+The Empty Qwen Image Layered Latent node creates a blank, multi-layered latent for the Qwen-Image-Layered model. It generates a tensor filled with zeros, sized for a batch, a stack of layers, and the spatial dimensions you request. This empty latent is the starting point of a layered generation, in the same way an empty latent image is the starting point of an ordinary one.
```
[the remark this answers is not in the document]
Comment (christian-byrne) [reply comfyui_embedded_docs/docs/EmptyQwenImageLayeredLatentImage/en.md]: Adopted in 720e74b0e. Reworked the whole page around a stack of tracing sheets and dropped `latent`, `tensor`, `temporal axis`, `z-index` and `metadata` from the prose.

Kept two things you suggested moving:

- **`LatentCutToBatch` with `dim` set to `t`.** It is now in a "Usage suggestions" section rather than the explanation. Without it the reader physically cannot get the layers out of this node's output, so removing it would leave the page accurate but unusable.
- **The `layers + 1` shape in the Outputs table.** That row was already in the shipped doc and is the thing the reader is most likely to be confused by, so it stays as the precise statement that the plain-language sections explain.

Alpha is now phrased as "the layers come out with transparency".

**Comment by christian-byrne** [comment]
Heads up on CI: GitHub Actions is in a declared major outage right now (started 15:22 UTC, webhook triggers throttled so push and PR events are not starting workflows). This repo's last Actions run was 2026-08-04, and `Comfy-Org/docs` last ran at 16:09 UTC today, while the app-based checks here (Socket, CodeRabbit) report within seconds on the same commits.

Practical effect: **CLA Assistant has not run, and `cla-assistant` is a required status check, so this PR shows BLOCKED for a reason that has nothing to do with its contents.** It should clear on its own once Actions recovers. Everything that can run has passed.

**Comment by christian-byrne** [comment]
recheck
